### PR TITLE
Fixes #56 by updating material icons to 3.0.1 release

### LINKED ISSUE #56: Material Design Icons 2.2.0 is out
Material design icons 2.2.0 has been updated with 41 new icons.
Material design icons: https://github.com/google/material-design-icons/releases

Thanks. :-)

### PR DESCRIPTION
**Pull request by bennyp123**
Steps I took:

```bash
# Fork
git clone git@github.com:bennyp123/iron-icons
cd iron-icons
sh ./update-icons.sh
git commit -am "Fixes #56 by updating material icons to 3.0.1 release"
git push -u origin master
# Submit PR

### CODE REVIEW

**Comment by bennyp123** [comment]
update-icons.sh:27-36

```bash
header() {
cat > $1 <<ENDL
<!--
Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
Code distributed by Google as part of the polymer project is also
subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
-->
```

should i replace the `@license` line?

**Comment by bennyp123** [comment]
https://github.com/PolymerElements/iron-icons/pull/65/commits/551f1c44c20df1d4299f04825b4aba0920701c48 replaces `@license` string

**Comment by notwaldorf** [comment]
@bennyp123 Hiya! Just merged the license string into master -- would you mind rebasing and regenerating the files, please?

**Comment by bennyp123** [comment]
sorry for the mess. had a file in the working dir that didn't belong when i rebased and commit. everything should be kosher now

**Comment by notwaldorf** [comment]
Hmm I'm not sure the rebase worked -- it looks like it's trying to make changes to `update-icons.sh` (which are on master) :(

File: iron-icons.html [hunk 1]
```diff
@@ -23,14 +23,10 @@
 
     <iron-icon icon="maps:directions-bus"></iron-icon>
 
-To load a subset of icons from one of the default `iron-icons` sets, you can
```
Comment (notwaldorf) [inline iron-icons.html]: Ugh, I realize the script removes this but can you maybe manually undo these changes?

File: update-icons.sh [hunk 2]
```diff
@@ -27,6 +27,7 @@ bootstrap() {
 header() {
 cat > $1 <<ENDL
 <!--
+@license
```
Comment (notwaldorf) [inline update-icons.sh]: It looks the rebase didn't work correctly? These should be on master already

**Comment by bennyp123** [comment]
I'm closing this and making a new PR.

**Comment by notwaldorf** [comment]
ha. sorry!
